### ISSUE TITLE
Feature request: % of Secrets discovered on the missions list

### ISSUE THREAD

**Issue by Irv1n3** [luciusDXL/TheForceEngine#162]
Hi, first thanks for your work during all those years!

On the Missions list, the only information recorded by the original game is the available missions and on what skill level the mission has been beat, I'm wondering if it could be also possible to get the highest % of discovered secrets displayed next to each mission.

Cheers

**Comment by NightFright2k19** [comment]
Isn't there a secrets counter on the objectives display in your PDA?

**Comment by Irv1n3** [comment]
Indeed, but I would like to keep record of them from the mission list!

**Comment by luciusDXL** [comment]
That makes sense, it would be nice to know so you can replay later to find more secrets.

**Comment by ifeldshteyn** [comment]
What would the counter look like ? Ex: 9 / 10 ? Where would you stick it?

![image](https://github.com/user-attachments/assets/39f3de3a-2b59-4761-9fa3-82b209fdf44e)
